### ISSUE TITLE
Fatal error - call to member function

### ISSUE THREAD

**Issue by katmoody** [humanmade/backupwordpress#972]
This person lost access to their site after activating the plugin: 

```
I just installed your plugin and it showed that it was installed successfully. But as soon as I pushed the Activate Plugin link it went to a blank page and gave me this error code:

Fatal error: Call to a member function set_archive_filename() on null in /home/btvsadmin/brandtuitives.com/wp-content/plugins/backupwordpress/functions/core.php on line 110
```

This is on Intercom ticket #https://app.intercom.io/a/apps/7f1l4qyq/inbox/all/conversations/1777144252

**Comment by pdewouters** [comment]
This is only supposed to run if they are upgrading from an older 1.x version - could you ask if they ever installed the plugin on that site before?

**Comment by willmot** [comment]
Fixed in #974
